src/index.js

```diff
@@ -0,0 +1,78 @@
+import "./style.css";
+import TaskStorage from "../module/TaskStorage.js";
+import UI from "../module/UI.js";
+import ManageTask from "../module/ManageTask.js";
+
+UI.displayTodoList();
+ManageTask.fillCheckBox();
+ManageTask.addEventToCheckBox();
+ManageTask.cleanAll();
+
+const toDo = document.querySelector(".form");
+toDo.addEventListener("change", () => {
+  const taskInput = document.getElementById("todo-box").value;
+  TaskStorage.addNewTask(taskInput);
+  window.location.reload();
+});
+
+const lists = document.querySelectorAll(".list");
+lists.forEach((list) => {
+  list.addEventListener("click", () => {
+    const form = list.children[1];
+    const inputBox = form.children[0];
+    const deleteIcon = form.children[1];
+    const menuIcon = form.children[2];
+    deleteIcon.style.display = "block";
+    menuIcon.style.display = "none";
+    list.style.backgroundColor = "#f2f2f2";
```

Comment:
This background color is repeated four times ins your file.
I would suggest you to create one item that will be having this background-color and call it only once.

Reply:
@GloireIrenge  Thank you for your feedback